src/u-root.md

```diff
@@ -46,416 +36,71 @@ attacks against shell interpreters[^11] and C programs[^8], we have started to
 look at using a more secure, modern language in embedded root file systems,
 namely, Go[^21] [^16].
 
-Go is a new systems programming language created by Google. Go has strong
-typing; language level support for concurrency; inter-process communication via
-channels, a la Occam[^13], Limbo[^17], and Alef[^27]; runtime type safety and
-other protective measures; dynamic allocation and garbage collection; closures;
-and a package syntax, similar to Java, that makes it easy to determine what
-packages a given program needs. The modern language constructs make Go a much
-safer language than C. This safety is critical for network-attached embedded
-systems, which usually have network utilities written in C, including web
-servers, network servers including sshd, and programs that provide access to a
-command interpreter, itself written in C. All are proving to be vulnerable to
-the attack-rich environment that the Internet has become. Buffer overflow
-attacks affecting C-based firmware code (among other things) in 2015 include
-GHOST and the so-called FSVariable.c bug in Intel’s UEFI firmware. Buffer
-overflows in Intel’s UEFI and Active Management Technology (AMT) have also been
-discovered in several versions in recent years.
-
-Both UEFI[^12] and AMT[^4] are embedded operating systems, loaded from flash that
-run network-facing software. Attacks against UEFI have been extensively
-studied[^9]. Most printers are network-attached and are a very popular
+## Safety and security
```

Comment:
Should this be in `# Benefits of using Go for firmware`?

Reply:
Yea I thought about, and then again, this paragraph picks up this idea of "embedded systems" and is mostly generic.
I would rather, eventually, want to make this a dedicated chapter, and reference that from both a Go perspective (here) and possibly a Rust perspective at some point.